inference-engine/src/plugin_api/debug.h

```diff
@@ -25,14 +25,16 @@
 
 #include "ie_algorithm.hpp"
 
+namespace InferenceEngine {
+namespace details {
+
```

Comment:
@ilya-lavrenov @apankratovantonp 

Please take a look at changes at this file. For explanation see commit message. Thanks